### ISSUE TITLE
person editor is slow

### ISSUE THREAD

**Issue by valearna** [WormBase/caltech-curation-services#37]
https://tazendra.caltech.edu/~postgres/cgi-bin/cecilia/person_editor.cgi
https://caltech-curation.textpressolab.com:4432/priv/cgi-bin/cecilia/person_editor.cgi

WBPaperID : 00060000 
click "Search Paper"

The dockerized form is much slower than tazendra. We need to figure out why and we need to make it faster

**Comment by valearna** [comment]
b941e4f about person_editor.cgi has the comment :
comments for Valerio to see where things are slower on dockerized vs tazendra.  against paper 00060000 &getAkaHash(); takes 3 secs, the loop before MID5 that does &matchFullnameToAka takes 5 seconds

**Comment by valearna** [comment]
I measured the time for each sub-function and it seems that the form is generally slower on docker but I can't pinpoint a specific part of the code that is slower than the others. DB queries are fast, and in-memory operations take most of the time but there's no specific function that takes up more time than others. I tried using the form on tazendra and it is a bit faster but I didn't find it much faster. Maybe the dockerized version is fine for now and we can try to improve performances later if we have cases where the difference between tazendra and docker is more significant? We should also see how it goes after deploying it to AWS
